js_modules/dagit/packages/core/src/workers/runLogs/runLogs.tsx

```diff
@@ -0,0 +1,51 @@
+import {ApolloClient} from '@apollo/client';
+
+import {PIPELINE_RUN_LOGS_SUBSCRIPTION} from '../../runs/PipelineRunLogsSubscription';
+import {PipelineRunLogsSubscription} from '../../runs/types/PipelineRunLogsSubscription';
+
+export type Message = INITIALIZE;
+type INITIALIZE = {
```

Comment:
`type InitializeMessage`? It's a little confusing on the eyes for a type to be `ALL_CAPS` since it looks like a constant value at a glance.